signer/src/stacks/contracts.rs

```diff
@@ -1466,7 +1466,7 @@ impl RotateKeysErrorMsg {
 /// A wrapper type for smart contract deployment that implements
 /// AsTxPayload.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, strum::Display)]
-#[strum(serialize_all = "snake_case")]
+#[strum(serialize_all = "kebab_case")]
```

Comment:
nit: `kebab-case` (according to https://docs.rs/strum/latest/strum/additional_attributes/index.html)

Reply:
I thought I changed that. I guess I just missed it.

Just an FYI, they both work, I noticed this mistake in some other struct and thought that it shouldn't work, but then I found https://github.com/Peternator7/strum/blob/v0.26.3/strum_macros/src/helpers/case_style.rs#L68. But yeah, I did intend to change it, and still do, I just missed it.